app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -17,41 +17,48 @@ $container-max-widths: (
   .nav-pills {
     background-color: var(--stanford-10-black);
     border-radius: var(--bs-nav-pills-border-radius);
+    width: 246px;
```

Comment:
Will using hard-coded widths cause issues for smaller viewports? I don't think too many users will be depositing files via their smartphones, but I guess the question, generally, is how much do we care about responsive design?

Reply:
@astridu ?

Reply:
yes, we care about responsive designs. I don't think it's a primary use case, but I'm sure someone is going to try to deposit on their tablet or phone. Is there a way to specify a max width for desktop without causing problems for the responsive designs?

Reply:
The responsive design is borked.

<img width="471" alt="image" src="https://github.com/user-attachments/assets/5a90e4f1-68ad-48f8-aed0-40fb30ba43bb" />

I think what you would want to do is pick an appropriate number of columns at each breakpoint that gives you an appropriately sized left nav. However, the left nav doesn't make sense once it has gotten small enough to stack, so we'll need some other sort of a nav (say, a hamburger).